.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "rules": {
         "react/no-unescaped-entities": "off",
         "react/prop-types": "off",
-        "no-const-assign": "off"
+        "no-const-assign": "off",
+        "react/jsx-key": "off"
```

Comment:
This should be enabled again. Without a correct `key` it's totally random what will happen to arrays that are mapped into React components, etc. Components might just not render, etc.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md

> If you are not using JSX then you can disable this rule.

We are :)